client/pages/new-review.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+export default class NewReview extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: '',
+      parkName: '',
+      isLoading: true
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      value: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const body = { parkCode: this.props.parkCode, userId: 1, content: this.state.value };
+    fetch('/api/reviews', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(body)
+    })
+      .catch(error => {
+        console.error('Error:', error);
+      });
+    location.hash = '#';
```

Comment:
It's best to wait until your promise becomes fulfilled before you transition to another screen.